firebase-appdistribution/firebase-appdistribution.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation project(':firebase-installations-interop')
     implementation project(':firebase-common')
     testImplementation project(path: ':firebase-appdistribution')
+    testImplementation project(':integ-testing')
```

Comment:
what is this? is it modern?

Reply:
This gives us access to `TestOnlyExecutors` so we can use the real Firebase executors in our tests.